Tests/Integration/IntegrationTestFixtures.swift

```diff
@@ -8,7 +8,7 @@
 import XCTest
 
 enum IntegrationTestFixtures {
-    static let network: NetworkPreset = .testNet
```

Comment:
Does it matter where this defaults?  not right now, right?

Reply:
not really